src/cdromfs.c

```diff
@@ -31,6 +31,8 @@ struct cdrom_dirent {
 
 static struct fs_volume *cdrom_volume_as_volume(struct cdrom_volume *cdv);
 static struct fs_dirent *cdrom_dirent_as_dirent(struct cdrom_dirent *cdd);
+void make_lower(char * name);
```

Comment:
For general consistency with C standard libraries, call this `strtoupper` and `strtolower` and move them to `string.c`